base/doc/ltnews39.tex

```diff
@@ -200,6 +200,19 @@ \subsection{\pkg{verbatim}: \cs{verb} showed visible spaces}
 \githubissue{1160}
 
 
+\section{Changes to files in the \pkg{cyrillic} category}
+
+\subsection{Correct definition of \cs{k}}
+
+Ages ago, the encoding specific definitions for various accent
+commands have been altered to guard altering some parameter values
```

Comment:
<  have been
> were

Reply:
Maybe also:

... to guard against altering ...

Reply:
Or, even better: 

... were changed to to guard against altering ...

Reply:
to to? :-)

Reply:
altered (or even changed) :-)